mods/cnc/rules/structures.yaml

```diff
@@ -309,6 +336,18 @@ HAND:
 
 AFLD:
 	Inherits: ^BaseBuilding
+	HitShape:
```

Comment:
It's more work, I know, but i'm thinking for cases like this and the comm center it would make more sense and help balancing if we defined the hitbox based on the ground footprint, and then accounted for these bits sticking up by increasing the Z offset of the targetable position.